cvat-core/src/enums.ts

```diff
@@ -34,6 +34,11 @@ export enum JobState {
     REJECTED = 'rejected',
 }
 
+export enum JobType {
+    ANNOTATION = 'annotation',
```

Comment:
I would suggest renaming to "REGULAR"

Reply:
It's not totally clear, what "regular" means. Originally, it was "normal", which had from the same problem.